bin.src/write_star_truth.py

```diff
@@ -0,0 +1,38 @@
+import argparse
```

Comment:
Could you please add the line
```
#!/usr/bin/env python
```
at the top of this file and in `write_gal_truth.py`.  The SCons build files are expecting that line when it makes an executable copy in the `bin` folder.